src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.js

```diff
@@ -19,6 +20,20 @@ function extractAttachmentsFromReport(report, reportActions) {
 
     const htmlParser = new HtmlParser({
         onopentag: (name, attribs) => {
+            const isVideo = Boolean(Str.isVideo(attribs['data-expensify-source'] || ''));
```

Comment:
This is temporary implementation to work with the current backend responses. We are waiting for new solutions connected to video player. After they are added we will totally change this function since video will be returned as `<video>` tags and with few new props